scapy/config.py

```diff
@@ -942,12 +946,14 @@ def crypto_validator(func):
     This a decorator to be used for any method relying on the cryptography library.  # noqa: E501
     Its behaviour depends on the 'crypto_valid' attribute of the global 'conf'.
     """
+
     def func_in(*args, **kwargs):
         # type: (*Any, **Any) -> Any
         if not conf.crypto_valid:
             raise ImportError("Cannot execute crypto-related method! "
                               "Please install python-cryptography v1.7 or later.")  # noqa: E501
         return func(*args, **kwargs)
+
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok